releasenotes/notes/chatgpt-llm-generator-d043532654efe684.yaml

```diff
@@ -0,0 +1,2 @@
+preview:
+  - Introduce `ChatGPTGenerator`, a class that can generate completions using OpenAI Chat models like ChatGPT and GPT4.
```

Comment:
```suggestion
  - Introduce `GPT35Generator`, a class that can generate completions using OpenAI Chat models like ChatGPT and GPT4.
```

The last place, I think :)

Reply:
Argh I never get them all!! :smile: Thank you :pray: